core/templates/newspapers.html

```diff
@@ -25,6 +25,7 @@
         <thead>
         <tr>
             <th class="headerSortDown left_no_border">State</th>
```

Comment:
OHHHH, I see - differentiating presentation order from sort.  I know there are tablesort JS libs that can do that, but let's just leave it how you have it for now and add a ticket for the idea of a more complicated system.  Definitely not this sprint.